json_serializable/test/generic_files/generic_class.g.dart

```diff
@@ -39,27 +39,42 @@ GenericClassWithConverter<T, S>
           ..fieldObject = json['fieldObject']
           ..fieldDynamic = json['fieldDynamic']
           ..fieldInt = json['fieldInt'] as int?
-          ..fieldT = _SimpleConverter<T?>()
-              .fromJson(json['fieldT'] as Map<String, dynamic>)
-          ..fieldS = _SimpleConverter<S?>()
-              .fromJson(json['fieldS'] as Map<String, dynamic>)
-          ..duration = const _DurationMillisecondConverter.named()
-              .fromJson(json['duration'] as int?)
-          ..listDuration = const _DurationListMillisecondConverter()
-              .fromJson(json['listDuration'] as int?);
+          ..fieldT = () {
```

Comment:
Ugh. This looks pretty ugly here.

Maybe we should generate a `_$SomeKindOfHelper` top-level function?

Reply:
Would this involve sharing the helper if used multiple time? If so, I don't see an obvious solution for it.

Reply:
There's a feature for that...to only generate a thing once.

Reply:
Could you give me a hint on how to do that? I see `context.addMember`, but it doesn't seem to generate only once.

Reply:
addMember SHOULD only add once if the content is identical.

Reply:
Done, that should be better now.